custom_metric.py

```
@@ -0,0 +1,329 @@
+    initialize_model,
+)
+from deepeval.test_case import (
+    LLMTestCase,
+    LLMTestCaseParams,
+    ConversationalTestCase,
+)
+from deepeval.metrics import BaseMetric
+from deepeval.models import DeepEvalBaseLLM
+from deepeval.metrics.contextual_recall.template import ContextualRecallTemplate
+from deepeval.metrics.indicator import metric_progress_indicator
+from deepeval.metrics.contextual_recall.schema import *
```

Ruff (0.8.2)

`from deepeval.metrics.contextual_recall.schema import *` used; unable to detect undefined names

(F403)